### DISCUSSION TITLE
为目录中的所有章节标题加上引导线

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#101] [Q&A] [answered]
目前目录页（目次页）仅2级标题和3级标题有引导线（section subsection）。希望能为一级标题（chapter）也加上引导线（还包括目录，插图目录，表格目录，参考文献，致谢）

**Comment by atxy-blip** [comment, marked as answer]
似乎在哪里回答过一遍这个问题：<https://github.com/nju-lug/NJUThesis/issues/66>

没有引导线是 `book` 文档类的默认配置，一般需要引入额外宏包做修改。
